### ISSUE TITLE
`xml:id` should be used in example 6

### ISSUE THREAD

**Issue by iherman** [w3c/sync-media-pub#53]
The [content model](https://w3c.github.io/sync-media-pub/sync-media.html#content-model) section refers to `xml:id` as the base for identification, but Example 6 uses `id`. That should be changed for `xml:id`.

(I may have missed other examples where the same problem occurs.)

**Comment by iherman** [comment]
Note, however, that the [EPUB media overlay](https://www.w3.org/TR/epub-33/#sec-smil-smil-elem) uses `id` and not `xml:id`. Another alternative is to use `id` for out document, and forget about `xml:id`. That would mean to be compatible with the EPUB media overlay spec.

Note: the latest SMIL version [deprecates the `id` element](https://www.w3.org/TR/SMIL/smil-structure.html#q6) in favor of `xml:id`. "Deprecation" does not mean that it is invalid to use `id`. This means that:

- if we use `xml:id` then we are perfect SMIL 3 citizens but are incompatible with EPUB 3
- if we define and use `id` then we are not perfect SMIL 3 citizens but are compatible with EPUB 3

I.e., we cannot please both :-) Personally, I favor being compatible with EPUB 3, ie, to introduce `id` and drop the reference to `xml:id`.

**Comment by marisademeglio** [comment]
Good point, when I officially set the SyncMedia serialization format to XML, I was just trying to be a good citizen and do what other people did in their specs. I saw in TTML and SMIL that `xml:id` was used, rather than `id`. However, I am happy to just use `id`, to be more consistent with EPUB 3.
